src/components/Layout.tsx

```diff
@@ -1,436 +1,124 @@
 /** @jsx jsx */
-import { Amplitude } from '@amplitude/react-amplitude';
-import { QueryResult } from '@apollo/react-common';
-import { useQuery } from '@apollo/react-hooks';
-import { css, Global, jsx } from '@emotion/core';
-import * as Sentry from '@sentry/node';
-import { isBrowser } from '@unly/utils';
+import { Amplitude, LogOnMount } from '@amplitude/react-amplitude';
+import { css, jsx } from '@emotion/core';
 import { createLogger } from '@unly/utils-simple-logger';
 import classnames from 'classnames';
-import { ThemeProvider } from 'emotion-theming';
-import get from 'lodash.get';
-// eslint-disable-next-line @typescript-eslint/no-unused-vars,no-unused-vars
 import React from 'react';
-import { useTranslation } from 'react-i18next';
 import { Button } from 'reactstrap';
-
-import { NRN_DEFAULT_FONT, NRN_DEFAULT_SECONDARY_COLOR, NRN_DEFAULT_THEME } from '../constants';
-import { LAYOUT_QUERY } from '../gql/common/layoutQuery';
 import ErrorPage from '../pages/_error';
-import { Customer } from '../types/data/Customer';
-import { Theme } from '../types/data/Theme';
-import { LayoutProps } from '../types/LayoutProps';
-import { getValue, STRATEGY_DO_NOTHING } from '../utils/record';
+import { MultiversalAppBootstrapProps } from '../types/MultiversalAppBootstrapProps';
 import Footer from './Footer';
-import Loader from './Loader';
+import Head from './Head';
 import Nav from './Nav';
 
-const fileLabel = 'components/Layout';
+const fileLabel = 'components/MultiversalAppBootstrap';
 const logger = createLogger({
   label: fileLabel,
 });
 
+type Props = {} & MultiversalAppBootstrapProps;
+
 /**
- * Layout of the whole app. Acts as a wrapper that displays the whole thing (menus, page, etc.)
  *
- * @param {Props} props
- * @return {JSX.Element}
- * @constructor
+ * @param props
  */
-const Layout: React.FunctionComponent<Props> = (props: Props): JSX.Element => {
+const Layout: React.FunctionComponent<any> = (props): JSX.Element => {
```

Comment:
Better not use `any`

Reply:
Thanks! I had missed that, a few bugs there indeed.

Reply:
I just pushed a fix about that